sway/sway.5.scd

```diff
@@ -353,6 +353,14 @@ runtime.
 	*bindcode* [--whole-window] [--border] [--exclude-titlebar] [--release] [--locked] [--input-device=<device>] [--no-warn] <code> <command>
 	is also available for binding with key/button codes instead of key/button names.
 
+*unbindsym* [--whole-window] [--border] [--exclude-titlebar] [--release] [--input-device=<device>] <key combo>
```

Comment:
Can you move this down so it is in alphabetical order? _I know there are some entries that do not comply with this, but we try to keep them in order_